vowpalwabbit/options_boost_po.h

```diff
@@ -156,7 +158,7 @@ struct options_boost_po : public options_i
  private:
   std::map<std::string, std::shared_ptr<base_option>> m_options;
 
-  std::vector<std::string> m_command_line;
+  //std::vector<std::string> m_command_line;

```

Comment:
Is this move required?

Reply:
Not required, as explained in the description this is a hack. I need to get the original command line so I can copy the options object; currently you can't get any unprocessed options out to use in the serializer, so I resorted to pulling out the original command line